dropwizard-client/src/main/java/io/dropwizard/client/DropwizardSSLConnectionSocketFactory.java

```diff
@@ -48,7 +55,12 @@ public SSLConnectionSocketFactory getSocketFactory() throws SSLInitializationExc
     private HostnameVerifier chooseHostnameVerifier() {
         HostnameVerifier hostnameVerifier;
         if (configuration.isVerifyHostname()) {
-            hostnameVerifier = SSLConnectionSocketFactory.getDefaultHostnameVerifier();
+            if (verifier == null) {
```

Comment:
What do you think about reducing the size of this method by using the ternary operator and several returns?
Something like that:
```java
if (configuration.isVerifyHostname()) {
    return verifier != null ? verifier : SSLConnectionSocketFactory.getDefaultHostnameVerifier();
} else {
    return new NoopHostnameVerifier();
}
```